### DISCUSSION TITLE
Md-text format checks: check for headings and lists syntax

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#11749] [Ideas]
It would be good if the markdown format will trigger a failure when heading or lists syntax ( `###` or `*`) does not appear on translation.

I have some placeholder checks for this:

`placeholders:r"^\* ":r"^#*\ "`

But maybe they should be part of the md-syntax checks.

**Comment by nijel** [comment]
Are you using native Weblate support for Markdown? In that case, the heading or lists syntax should not be present in a string.

**Reply by emmapeel2** [reply]
No, I am using a .po generator that reads some markdown files and also some templates in jinja2...

**Reply by nijel** [reply]
I don't think the block syntax should be exposed to the translator. That's what native support does, and I believe po4a behaves the same on this.
